src/test/ui/parser/issue-63135.stderr

```diff
@@ -23,16 +23,16 @@ LL | fn i(n{...,f #
    |        `..` must be at the end and cannot have a trailing comma
 
 error: expected `[`, found `}`
-  --> $DIR/issue-63135.rs:3:15
+  --> $DIR/issue-63135.rs:3:16
    |
 LL | fn i(n{...,f #
-   |               ^ expected `[`
+   |                ^ expected `[`
```

Comment:
May I ask is the extra space expected behavior? I wasn't sure which caused this extra space.

Reply:
I'm not sure, perhaps lexer considers the newline a one more character (so span of the recovered closing delimiter is at EOF after the newline), but error rendering still puts the label on the same line?